charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/nodes-darwin.yaml

```diff
@@ -4,7 +4,7 @@ Do not change in-place! In order to change this file first read following link:
 https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack/hack
 */ -}}
 {{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
-{{- if and (or .Values.grafana.enabled .Values.grafana.forceDeployDashboards) (semverCompare ">=1.14.0-0" $kubeTargetVersion) (semverCompare "<9.9.9-9" $kubeTargetVersion) .Values.grafana.defaultDashboardsEnabled .Values.nodeExporter.enabled .Values.nodeExporter.operatingSystems.darwin.enabled }}
```

Comment:
@TheKangaroo did you change this by hand or using the script?

Reply:
I think this is coming from https://github.com/prometheus-community/helm-charts/blob/723cfad365d5c55b84162d4a5f11294ea736ca54/charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py#L57. Could you replace the conditions by adding  `.Values.nodeExporter.enabled ` before the operating system conditions and run the sync script one last time?

Reply:
I think it was changed by the script, but i can revert the change on the non windows rules

Reply:
I'd rather it's rolled back yes

Reply:
I rebuild against latest main (kube-prometheus-stack and kube-prometheus) and added only changes to windows dashboards and rules. should be fine now